Makefile

```
@@ -87,6 +89,18 @@ check-fmt: build ## Verify formatting
 lint: check-architecture ## Run linters
 	$(UV_ENV) $(UV) run ruff check
 	$(PYLINT) $(PYLINT_TARGETS)
+	$(SKYLOS) $(SKYLOS_TARGETS) --category dead_code --gate --format concise --no-upload --no-provenance
+
+skylos-allow: ## Document one intentional Skylos finding
```

checkmake (0.3.2)

[warning] Target body for "skylos-allow" exceeds allowed length of 5 lines (9).

(maxbodylength)